modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIApplicationRelevantObjectEntryModelListener.java

```diff
@@ -80,12 +81,25 @@ private void _validate(ObjectEntry objectEntry) {
 					_objectFieldLocalService.getObjectField(
 						objectEntry.getObjectDefinitionId(), "baseURL");
 
+				String message = null;
+				String messageKey = null;
+
+				if (!StringUtil.isLowerCase(baseURL)) {
+					message = "%s must contain only lowercase characters.";
```

Comment:
The ending `.` is not consisten with the message in the `else` path. Should we remove it?

Reply:
Hmmm It's strange. If you take a look at the language.properties the message `x-can-have-a-maximum-of-255-alphanumeric-characters` uses a `.` at the end too. So what seems to be inconsistent is the message that was present before. I will change it and send it again!